easybuild/easyconfigs/c/cuDNN/cuDNN-8.0.4.30-CUDA-11.0.2.eb

```diff
@@ -1,30 +1,29 @@
 ##
-# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
-#
-# Author:    Stephane Thiell <sthiell@stanford.edu>
```

Comment:
@robert-mijakovic It's fine if you add yourself as author, but keep the original one too please.

Reply:
I'm sorry about it.  It wasn't intentional. I will fix it.